v3data/__init__.py

```diff
@@ -29,8 +36,32 @@ async def query(self, query: str, variables=None) -> dict:
             params = {"query": query, "variables": variables}
         else:
             params = {"query": query}
+        # TODO: error handling -> connection, result and others   httpcore.RemoteProtocolError
+        #                                                         ssl.SSLError:   [SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure
+        #
         response = await async_client.post(self._url, json=params)
-        return response.json()
+
+        if response.status_code == 200:
+            if parsed_response := response.json():
```

Comment:
Actually this will fail if response is not valid json right?
think we might need a try except here.